classification1.Rmd

```diff
@@ -846,71 +991,88 @@ The new imbalanced data is shown in Figure \@ref(fig:05-unbalanced).
 set.seed(3)
 rare_cancer <- bind_rows(
     filter(cancer, Class == "B"),
-    cancer |> filter(Class == "M") |> slice_sample(n = 3)
+    cancer |> 
+      filter(Class == "M") |> slice_sample(n = 3)
     ) |>
   select(Class, Perimeter, Concavity)
 
 rare_plot <- rare_cancer |>
   ggplot(aes(x = Perimeter, y = Concavity, color = Class)) +
   geom_point(alpha = 0.5) +
-  labs(color = "Diagnosis") +
+  labs(color = "Diagnosis", x = "Perimeter (standardized)", y = "Concavity (standardized)") +
   scale_color_manual(labels = c("Malignant", "Benign"), values = c("orange2", "steelblue2"))
 rare_plot
 ```
 > Note: You will see in the code above that we use the `set.seed` function.
 > This is because we are using `slice_sample` to artificially pick only 3 of
-> the malignant tumour observations, which uses random sampling to choose which
+> the malignant tumor observations, which uses random sampling to choose which
 > rows will be in the training set. In order to make the code reproducible, we
 > use `set.seed` to specify where the random number generator starts for this
 > process, which then guarantees the same result, i.e., the same choice of 3
 > observations, each time the code is run. In general, when your code involves
 > random numbers, if you want *the same result* each time, you should use
 > `set.seed`; if you want a *different result* each time, you should not.
```

Comment:
maybe add a reminder to this note that in your own data analysis, you only set.seed once at the top of the analysis (so that the rest of the analysis uses numbers that seem to be random)